### PR TITLE
Answered question  sum2.py

### PR DESCRIPTION
**Pull request by Claire56**
answered the question on file sum2.py

### CODE REVIEW

**Comment by Claire56** [comment]
Hi Sergey ,please review my code, thanks

File: sum2.py [hunk 1]
```diff
@@ -10,7 +10,17 @@
...
 
-def sum2(nums):
+def sum2(nums,t):
     # todo: fill the bpdy of the function
-    pass
+    my_dict = {}
+    for i,value in enumerate(nums):
+        needed = t - value
+        if needed in my_dict:
+            result = (my_dict[needed],i)
```
Comment (shevelevs) [inline sum2.py:19]: you can combine the two lines, i.e.
```
return my_dict[needed], i
```

File: sum2.py [hunk 2]
```diff
@@ -10,7 +10,17 @@
...
+            return result
+        else:
+            my_dict[value]= i
 
+
+
+sum2([2,7,11,15],9)
```
Comment (shevelevs) [inline sum2.py:26]: Do you think a single test case is enough? Think about
* tests that demonstrate that your code works according to the problem statement
* edge cases (what could they be here?)

**Review by shevelevs — COMMENTED** [review]
Solution looks good, but you could use more test cases. And thanks for making this easier for me to review and comment on!

File: sum2.py (hunk 1 above)
Comment (Claire56) [reply sum2.py:19]: Ok, thanks
